src/include/liburing.h

```diff
@@ -34,6 +34,10 @@
 #define uring_likely(cond)	__builtin_expect(!!(cond), 1)
 #endif
 
+#ifndef LIB_URING_STATIC_INLINE
+#define LIB_URING_STATIC_INLINE static inline
```

Comment:
Can we calls this something a bit more condensed? It looks like just search/replace was done, which means we now have lines that are > 80 chars and the careful aligning that was otherwise done now just looks like a mess.

Reply:
I apologize for this.

I'm used to more verbose macros.

We wouldn't need to do anything if we simply used `STATIC_INLINE`. Same characters and length.

This also reminds me, I didn't `#undef` the macro at the end of the header. Not best practices.

Reply:
Both of those sound good.